prow/plugins/plugins.go

```diff
@@ -131,14 +131,32 @@ type PluginAgent struct {
 // target for plugin Configuration
 type Configuration struct {
 	// Repo (eg "k/k") -> list of handler names.
-	Plugins         map[string][]string `json:"plugins,omitempty"`
-	Triggers        []Trigger           `json:"triggers,omitempty"`
-	Heart           Heart               `json:"heart,omitempty"`
-	MilestoneStatus MilestoneStatus     `json:"milestonestatus,omitempty"`
-	Slack           Slack               `json:"slack,omitempty"`
-	// ConfigUpdater holds config for the config-updater plugin.
-	ConfigUpdater ConfigUpdater `json:"config_updater,omitempty"`
-	Blockades     []Blockade    `json:"blockades,omitempty"`
+	Plugins map[string][]string `json:"plugins,omitempty"`
+
+	// ExternalPlugins is a list of external plugin names mapped
+	// to Kubernetes Service names.
+	ExternalPlugins []ExternalPlugin `json:"external_plugins"`
```

Comment:
Should we do `map[string][]ExternalPlugin` to match `Plugins`?

Reply:
Sounds good. Solves my concern about mapping these plugins to repos.